<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>973</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Professional CFD Tool</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QMenuBar" name="menuBar">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <addaction name="menuFile"/>
     <addaction name="menuEdit"/>
     <addaction name="menuView"/>
     <addaction name="menuHelp"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="middleLayout">
     <item>
      <layout class="QVBoxLayout" name="leftLayout">
       <item>
        <widget class="QPushButton" name="LoadGeomButton">
         <property name="text">
          <string>Load Geometry</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="backgroundComboBoxLabel_2">
         <property name="text">
          <string>Setup Case</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="backgroundComboBoxLayout"/>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>122</width>
            <height>285</height>
           </rect>
          </property>
          <widget class="QTabWidget" name="tabWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>127</width>
             <height>291</height>
            </rect>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Tab 1</string>
            </attribute>
            <widget class="QLineEdit" name="lineEdit">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>10</y>
               <width>51</width>
               <height>21</height>
              </rect>
             </property>
            </widget>
            <widget class="QSpinBox" name="N_Layers">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>70</y>
               <width>51</width>
               <height>26</height>
              </rect>
             </property>
            </widget>
            <widget class="QCheckBox" name="checkBox">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>200</y>
               <width>121</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>Improve Mesh</string>
             </property>
            </widget>
            <widget class="QDialogButtonBox" name="buttonBox">
             <property name="geometry">
              <rect>
               <x>-10</x>
               <y>230</y>
               <width>166</width>
               <height>25</height>
              </rect>
             </property>
             <property name="standardButtons">
              <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
             </property>
            </widget>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>41</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>min_x</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>40</y>
               <width>51</width>
               <height>21</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>40</y>
               <width>51</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>max_x</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>70</y>
               <width>51</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>Layers</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Tab 2</string>
            </attribute>
           </widget>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="leftSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QOpenGLWidget" name="openGLWidget">
       <property name="minimumSize">
        <size>
         <width>1000</width>
         <height>700</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout">
     <item>
      <spacer name="bottomSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="backgroundComboBoxLabel">
       <property name="text">
        <string>Background</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="backgroundComboBox">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="topViewButton">
       <property name="text">
        <string>Top View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="frontViewButton">
       <property name="text">
        <string>Front View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sideViewButton">
       <property name="text">
        <string>Side View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fitViewButton">
       <property name="text">
        <string>Fit to View</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>152</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>152</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
  </layout>
  <action name="menuFile">
   <property name="text">
    <string>&amp;File</string>
   </property>
  </action>
  <action name="menuEdit">
   <property name="text">
    <string>&amp;Edit</string>
   </property>
  </action>
  <action name="menuView">
   <property name="text">
    <string>&amp;View</string>
   </property>
  </action>
  <action name="menuHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
